influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -358,17 +373,44 @@ pub(crate) fn parse_validate_and_update_schema(
     schema: &DatabaseSchema,
     partitioner: &Partitioner,
     default_time: i64,
+    accept_partial: bool,
+    precision: Precision,
 ) -> Result<ValidationResult> {
```

Comment:
This function (`parse_validate_and_update_schema`) as well as the functions it calls:

* `validate_or_insert_schema_and_partitions`, which calls
* `validate_and_convert_parsed_line`

Do not look like they need to return `Result`. Looks like this predates this PR, and perhaps some day they will return `Result`, as `validate_and_convert_parsed_line` uses `assert!` and therefore may panic; so, not going to block on this, but thought it worth mentioning.

Reply:
I think my changes to detecting limits will end up throwing an error here eventually so I think we don't need to worry about it really!